addons/binding/org.openhab.binding.ihc/src/main/java/org/openhab/binding/ihc/internal/ChannelUtils.java

```diff
@@ -0,0 +1,233 @@
+/**
+ * Copyright (c) 2010-2018 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ihc.internal;
+
+import static org.openhab.binding.ihc.internal.IhcBindingConstants.*;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.binding.builder.ChannelBuilder;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
+import org.openhab.binding.ihc.internal.config.ChannelParams;
+import org.openhab.binding.ihc.internal.handler.IhcHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+
+/**
+ * Generic methods related to openHAB channels.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public class ChannelUtils {
+    private static final Logger LOGGER = LoggerFactory.getLogger(IhcHandler.class);
```

Comment:
Loggers should only be final, not static.

Reply:
methods which use this logger are static